istio/src/main/java/com/alibaba/nacos/istio/util/IstioCrdUtil.java

```diff
@@ -78,6 +78,11 @@ public static ServiceEntryWrapper buildServiceEntry(String serviceName, IstioSer
                 }
             }
 
+            String metaHostname = instance.getMetadata().get(ISTIO_HOSTNAME);
+            if (StringUtils.isNotEmpty(metaHostname)) {
```

Comment:
remove

Reply:
Allow the service to customize the name synced to istio. Use its metadata.

Reply:
Understand. Please fix code format.
And execute this command.
```
mvn -Prelease-nacos clean install -Dmaven.test.skip=true
```